File: Atividade 5 V1/Quest6.pas
Program IdadeAtleta ;
    var idade:integer;
        Begin
	       write('Informe a idade da pessoal= ');
	       readln(idade);
	       if (((idade)>=10) and ((idade)<=12)) then
	           begin
			     write('Infantil');
	           end
	       else if (((idade)>12) and ((idade)<=18)) then
		      begin
			     write('Juvenil');
			 end
		  else if (((idade)>18) and ((idade)<=40)) then
		      begin
			     write('Adulto');
			 end
		  else
		      begin
			     write('Não é atleta');
			 end	   
	   End.

File: Alg2-A1/Quest6.c
#include <stdio.h>

void main(){
	int idade;
	printf("Informe o valor da idade= ");
	scanf("%d",&idade);
	if(10<=idade&12>=idade){
		printf("Infantil");
	}else if(12<idade&18>=idade){
		printf("Juvenil");
	}else if(18<idade&40>=idade){
		printf("Adulto");
	}else if(10>idade|40<idade){
		printf("Nao e atleta");
	}
}
